templates/install_puppet.bat.erb

```diff
@@ -2,6 +2,9 @@ set AGENT_PID=%1
 set windowTitle=Puppet Agent Upgrade
 title %windowTitle%
 
+REM set code page to US (English) to adjust non-US English systems if required
+chcp 437
```

Comment:
For future reference, this can cause some unintended consequences as Ruby will set `Encoding.default_external` to this value.  It's also a global setting in Windows, which will impact other applications.

We sometimes use this in tests, but typically revert it back immediately.

In any event, we've found a simpler method of checking for running processes in the other PR.

Thanks again!